Collection Interfaces/List Interface/Solution.java

```diff
@@ -0,0 +1,34 @@
+import java.util.*;
```

Comment:
(in line 2-3)
add documentation(in form of comment explaining [ isEmpty() method ] in 2-3 lines.
refer :  [this ](https://github.com/VBSquad/Scrabbling-Java-With-CF/blob/master/Collection%20Interfaces/List%20Interface/boolean%20isEmpty()/isEmptyFunc.java)

Reply:
for now add this : The remove(int index) method of List interface in Java is used to remove an element from the specified index from a List container and returns the element after removing it. It also shifts the elements after the removed element by 1 position to the left in the List.